src/WCSimDetectorConfigs.cc

```diff
@@ -40,7 +40,6 @@ void WCSimDetectorConstruction::SetSuperKGeometry()
   WCAddGd               = false;
 }
 
-
```

Comment:
Put this extra line back in so that this DetectorConfig file doesn't show up in the diff for this PR.